File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/infrastruktur/config/OidcConfiguration.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.infrastruktur.config;

import no.nav.security.token.support.core.configuration.ProxyAwareResourceRetriever;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class OidcConfiguration {
    @Bean
    public ProxyAwareResourceRetriever resourceRetriever() {
        return new ProxyAwareResourceRetriever();
    }
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/config/OidcConfiguration.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.config

import no.nav.security.token.support.core.configuration.ProxyAwareResourceRetriever
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
open class OidcConfiguration {
    @Bean
    open fun resourceRetriever(): ProxyAwareResourceRetriever {
        return ProxyAwareResourceRetriever()
    }
}
